resources/public/api/conf/1.0/application.yaml

```diff
@@ -331,6 +355,9 @@ components:
     LocalReferenceNumber:
       type: string
       pattern: '.{1,22}'
+    ConsigneeId:
+      type: string
+      pattern: '[0-9]{2}[A-Z]{2}[A-Z0-9]{16}[0-9]'
```

Comment:
Looking at the TraderIdType in the types-v16-0.xsd, the regex for the ConsigneeId should be `.{1,16}` I'm not sure why this is slightly different than the Excise Reference Number but we should probably match that in our API spec.

Reply:
done